openapi.go

```diff
@@ -425,12 +425,19 @@ func parseStructTags(t reflect.Type, schemaRef *openapi3.SchemaRef) {
 		example, ok := field.Tag.Lookup("example")
 		if ok {
 			propertyValue.Example = example
-			if propertyValue.Type.Is(openapi3.TypeInteger) {
+			switch {
+			case propertyValue.Type.Is(openapi3.TypeInteger):
 				exNum, err := strconv.Atoi(example)
```

Comment:
Here I will suggest you take a look at OpenAPI spec, could an integer be a uint so large that it exceed int.

If you have to deal with this

One common way is to check if the first character is an `-` (hyphen) then you can use Atoi that use ParseInt, and if there is no leading `-`, the string doesn't represent a negative number so it can fit a uint, so you can use ParseInt(s, 10, 64)

Reply:
Ah I see that is a good point.